.github/workflows/ci.yaml

```
@@ -0,0 +1,61 @@
+name: CI
+
+on:
+  push:
+    branches: [master, staging, trying]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```

We've mostly specified specific versions just to ensure there are no surprises when `-latest` next changes. Same deal applies to the deploy job below, though you could also make that a final and optional step in the build job, I don't mind either way.